test/suites/para/test_tanssi_containers.ts

```diff
@@ -394,7 +396,7 @@ async function countUniqueBlockAuthors(paraApi, blockStart, blockEnd, numAuthors
 
   let uniq = [...new Set(actualAuthors)];
 
-  if (uniq.length != numAuthors) {
+  if (uniq.length <= numAuthors) {
```

Comment:
where are we checking that at least we have 2 authors?

Reply:
I mean we should check that numAuthors > 1 as well right?

Reply:
We could, but in one test we check that 5 consecutive blocks have 2 authors, so there will be a small probability of failure if one of them fails to produce 2 or 3 blocks in a row.

Reply:
Also I put the wrong `if` condition, it should be negated

Reply:
I'd say for now that is a very small probability, I would add the >1 author check. If we keep getting failures, we will evaluate, but at least we check that they take somewhat turns